inc/options/minify.php

```diff
@@ -81,504 +159,764 @@
+					<?php $this->checkbox( 'minify.html.inline.css', false, 'html_' ); ?> <?php Util_Ui::e_config_label( 'minify.html.inline.css' ); ?></label><br />
+					<?php $this->checkbox( 'minify.html.inline.js', false, 'html_' ); ?> <?php Util_Ui::e_config_label( 'minify.html.inline.js' ); ?></label><br />
+					<?php $this->checkbox( 'minify.html.reject.feed', false, 'html_' ); ?> <?php Util_Ui::e_config_label( 'minify.html.reject.feed' ); ?></label><br />
 					<?php
-$html_engine_file = '';
+					$html_engine_file = '';
 
-switch ( $html_engine ) {
-case 'html':
-case 'htmltidy':
-	$html_engine_file = W3TC_INC_DIR . '/options/minify/' . $html_engine . '.php';
-	break;
-}
+					switch ( $html_engine ) {
+						case 'html':
+						case 'htmltidy':
+							$html_engine_file = W3TC_INC_DIR . '/options/minify/' . $html_engine . '.php';
+							break;
+					}
 
-if ( file_exists( $html_engine_file ) ) {
-	include $html_engine_file;
-}
-?>
+					if ( file_exists( $html_engine_file ) ) {
+						include $html_engine_file;
+					}
+					?>
 				</td>
 			</tr>
 			<tr>
-				<th><label for="minify_html_comments_ignore"><?php Util_Ui::e_config_label( 'minify.html.comments.ignore' ) ?></label></th>
+				<th><label for="minify_html_comments_ignore"><?php Util_Ui::e_config_label( 'minify.html.comments.ignore' ); ?></label></th>
 				<td>
 					<textarea id="minify_html_comments_ignore"
-						<?php Util_Ui::sealing_disabled( 'minify.' ) ?>
+						<?php Util_Ui::sealing_disabled( 'minify.' ); ?>
 						name="minify__html__comments__ignore" class="html_enabled" cols="40" rows="5"><?php echo esc_textarea( implode( "\r\n", $this->_config->get_array( 'minify.html.comments.ignore' ) ) ); ?></textarea>
-					<p class="description"><?php _e( 'Do not remove comments that contain these terms.', 'w3-total-cache' ); ?></p>
+					<p class="description"><?php esc_html_e( 'Do not remove comments that contain these terms.', 'w3-total-cache' ); ?></p>
 				</td>
 			</tr>
 			<?php
-$html_engine_file2 = '';
+			$html_engine_file2 = '';
 
-switch ( $html_engine_file2 ) {
-case 'html':
-case 'htmltidy':
-	$html_engine_file = W3TC_INC_DIR . '/options/minify/' . $html_engine . '2.php';
-	break;
-}
+			switch ( $html_engine_file2 ) {
+				case 'html':
+				case 'htmltidy':
+					$html_engine_file = W3TC_INC_DIR . '/options/minify/' . $html_engine . '2.php';
+					break;
+			}
 
-if ( file_exists( $html_engine_file2 ) ) {
-	include $html_engine_file2;
-}
-?>
+			if ( file_exists( $html_engine_file2 ) ) {
+				include $html_engine_file2;
+			}
+			?>
 		</table>
 
 		<?php Util_Ui::button_config_save( 'minify_html_xml' ); ?>
 		<?php Util_Ui::postbox_footer(); ?>
 
-		<?php Util_Ui::postbox_header( __( '<acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ), '', 'js' ); ?>
+		<?php
+		Util_Ui::postbox_header(
+			sprintf(
+				// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag.
+				__(
+					'%1JS%2',
```

Comment:
PHP Fatal error:  Uncaught ValueError: Unknown format specifier "J".
Invalid placeholders -- Change `%1JS%2` to `%1$sJS%2$s`.

Reply:
Fixed placeholders